src/main/java/seedu/bankwithus/AccountList.java

```diff
@@ -1,14 +1,32 @@
 package seedu.bankwithus;
 
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class AccountList {
-
+    protected ArrayList<Account> accounts;
```

Comment:
Why... you might as well name it public since everything is in the same package...

Reply:
Ok I'll change it later!

Reply:
Lol wait no. I was writing these reviews as a joke. And plus I intention was for this to be changed from protected to private, not public

Reply:
Ok sure! We may need to standardise these in the next version.